about_first_project.css

```diff
@@ -0,0 +1,122 @@
+body {
+    background-color: mintcream;
+    margin: 0px; 
+    color: black;
+}
+
+h1 {
+    font-size: 50px;
+    font-family: 'Montserrat', sans-serif;
+    padding: 50px 10px;
+    margin: 15px 10px;
+    text-align: center;
+}
+h2 {
+    font-size: 30px;
+    color: black;
+    font-family: 'Nunito', sans-serif;
+    margin: 20px 30px;
+    font-style: oblique;
+    
+}
+    
+p {
+    font-size: 23px;
+    color: black;
+    font-family: 'Nunito', sans-serif;
+    margin: 0 30px;
+    text-align: justify;
+}
+
+
+
+/*** Header ***/
+header {
+    background: #C9F4EB;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    padding: 50px 16px;
+    background-image: url("./images/KH_header_backgr_a5_1.jpg");
+  }
+  
+  
+  .hamburger{
+    display: flex;
+    height: 30px;
+    width: 30px;
+    flex-direction: column;
+    justify-content: space-between;
+  }
```

Comment:
For this to stay in the top right corner, I would suggest you set the position to be absolute. Then you can decide exactly where you want it to be, and the position will not change:
  ```
position: absolute;
  top: 20px;
  right: 70px;
```

Reply:
And PS - a functioning dropdown would require some JavaScript :) Take a look [here](https://www.w3schools.com/howto/howto_js_dropdown.asp) if you want an example.

Reply:
Also, this piece of code doesn't have to be indented - it should be at the same level as the other code. Same goes for all the elements below, all the way to the media query.